rplugin/python3/iron/repls/php.py

```diff
@@ -0,0 +1,15 @@
+# encoding:utf-8
+"""PHP repl definition for iron.nvim. """
+from iron.repls.utils.cmd import detect_fn
+
+php = {
+    'command': 'php -a',
+    'language': 'php',
+    'detect': detect_fn('php'),
+}
+
+psyshell = {
+    'command': 'psysh',
+    'language': 'php',
+    'detect': detect_fn('php'),
```

Comment:
I added this above the php entry in available_repls because I assumed that the order there is what matters (sadly I did not find any of this documented in the code so I just had to go with trial and error and guess about the meaning of these dict entries).

If I start `:IronRepl` in a php file I get the psyshell repl.